.azure-pipelines/gpu-tests.yml

```diff
@@ -72,12 +70,15 @@ jobs:
     - bash: bash .actions/pull_legacy_checkpoints.sh
       displayName: 'Get legacy checkpoints'
 
-    - bash: |
-        python -m coverage run --source pytorch_lightning -m pytest pytorch_lightning tests --ignore tests/benchmarks -v --junitxml=$(Build.StagingDirectory)/test-results.xml --durations=50
-      displayName: 'Testing: standard'
+    - bash: python -m coverage run --source pytorch_lightning -m pytest pytorch_lightning
+      workingDirectory: src
+      displayName: 'Testing: doctests'
 
-    - bash: |
-        bash tests/standalone_tests.sh
+    - bash: python -m coverage run --source pytorch_lightning -m pytest tests --ignore tests/benchmarks -v --junitxml=$(Build.StagingDirectory)/test-results.xml --durations=50
+      displayName: 'Testing: unittests'
+
+    - bash: bash run_standalone_tests.sh
```

Comment:
This is not renamed in this PR

Reply:
Tests are not changed in this PR, or what do you mean?

Reply:
I mean that `run_standalone_tests` does not exist in this PR